python/eos/pyhf.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python
+# vim: set sw=4 sts=4 et tw=120 :
+
+# Copyright (c) 2020-2023 Danny van Dyk
```

Comment:
This needs Lorenz's copyright notice instead! Please remove mine or at least limit it to 2023.

Reply:
Also fixed.